src/main/kotlin/io/nais/NaisApplication.kt

```diff
@@ -71,8 +70,20 @@ data class Prometheus(
 
 @Serializable
 data class Resources(
+   val limits: Limits,
+   val requests: Requests,
+)
+
+@Serializable
+data class Requests(
+   val cpu: String,
+   val memory: String,
+)
+
+@Serializable
+data class Limits(
    val cpu: String,
-   val memory: String
+   val memory: String,
 )
 
```

Comment:
No need for two different classes with the same members

Reply:
Fix'd! =)